src/main/java/org/jenkinsci/plugins/saml/IdpMetadataConfiguration.java

```diff
@@ -179,7 +180,7 @@ public String getDisplayName() {
             return "";
         }
 
-        public FormValidation doTestIdpMetadata(@QueryParameter("xml") String xml) {
+        public FormValidation doTestIdpMetadata(@QueryParameter("xml") String xml) throws IOException {
```

Comment:
it should return a Fromvalidation with an error message, it never should throw an exception

Reply:
Thank you for the hint, I changed it